### PR TITLE
Simplify imports: #110

### PR DESCRIPTION
**Pull request by Okabe-Rintarou-0**
Add some imports in __init__.py to simplify import.

### CODE REVIEW

File: fastembed/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from .embedding import TextEmbedding, DefaultEmbedding, FlagEmbedding, JinaEmbedding
```
Comment (KShivendu) [inline fastembed/__init__.py:1]:
```suggestion
from .embedding import TextEmbedding, DefaultEmbedding, FlagEmbedding, JinaEmbedding

__all__ = [TextEmbedding, DefaultEmbedding, FlagEmbedding, JinaEmbedding]
```

Please run the formatter before pushing

**Review by KShivendu — COMMENTED** [review]
Cool. I think it's a valid suggestion. 

@NirantK wdyt?

**Comment by joein** [comment]
It’s not yet there, only TextEmbedding should be added, the others are outdated

**Comment by joein** [comment]
`embedding.py` is kept only for backward compatibility, I would prefer to have import from a proper path because we may delete `embedding.py` in future releases

thanks for the patience

**Review by NirantK — APPROVED** [review]

**Pull request merged**